tests/unit_tests/epee_utils.cpp

```diff
@@ -1751,65 +1751,89 @@ TEST(parsing, number)
   boost::string_ref val;
   std::string s;
   std::string::const_iterator i;
+  bool is_float_val;
```

Comment:
it's a bad idea to leave these 2 unassigned at declaration. It will behave deterministically in the debug mode (will be assigned to `false`), but nondeterministically in the release mode.

Since the expected result is that these become `false`, I suggest that they be assigned `true` at their declaration..